src/labels/CustomLabels.labels

```diff
@@ -4214,6 +4270,548 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
+    </labels>
+    <labels>
+        <fullName>geButtonBuilderNavBackFormFields</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Gift Entry Template Builder navigation button text</shortDescription>
+        <value>Back: Form Fields</value>
+    </labels>
+    <labels>
+        <fullName>geButtonBuilderNavBackTemplateInfo</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Gift Entry Template Builder navigation button text</shortDescription>
+        <value>Back: Template Info</value>
+    </labels>
+    <labels>
+        <fullName>geButtonBuilderNavBatchHeader</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Gift Entry Template Builder navigation button text</shortDescription>
+        <value>Next: Batch Header</value>
+    </labels>
+    <labels>
+        <fullName>geButtonBuilderNavFormFields</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Gift Entry Template Builder navigation button text</shortDescription>
+        <value>Next: Form Fields</value>
+    </labels>
+    <labels>
+        <fullName>geButtonFormFieldsAddSection</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button to add new section to template</shortDescription>
+        <value>Add Section</value>
+    </labels>
+    <labels>
+        <fullName>geButtonFormFieldsCollapseAll</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button to collapse all</shortDescription>
+        <value>Collapse All</value>
+    </labels>
+    <labels>
+        <fullName>geButtonFormFieldsExpandAll</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button to expand all</shortDescription>
+        <value>Expand All</value>
+    </labels>
+    <labels>
+        <fullName>geButtonFormFieldsModalDeleteSectionAndFields</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button to delete a section and remove its fields</shortDescription>
+        <value>Delete Section and Fields</value>
+    </labels>
+    <labels>
+        <fullName>geButtonSaveAndClose</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Gift Entry Template Builder save button text</shortDescription>
```

Comment:
@psadie-sfdo @markross-sfdo tagging you both because I'll be on PTO when you get to this comment. If it makes sense to update the label's dev name to be more general, this text will also need to be generalized.

Reply:
Following the pattern in the other 'common' labels in that the value and short description match.